### DISCUSSION TITLE
Clover crashes on HP XW8600 with legacy BIOS

### DISCUSSION THREAD

**Discussion by KevinMoorman** [CloverHackyColor/CloverBootloader#558] [Q&A] [answered]
As many others, I am desirous to boot from NVME M.2 mounted on PCI-E adapter.
If I boot Windows 10 from HDD, the SSD is detected, can be formatted, can transfer files back and forth, but of course does not show up as a bootable device from within the BIOS.

So I got "[boot disk utility](http://cvad-mac.narod.ru/index/bootdiskutility_exe/0-5)" and used it to format a 16GB Lexar USB flash drive with Clover R5149.
Then I went to this directory G:\EFI\CLOVER\drivers\off
and copied NvmExpressDxe.efi
Pasted it into this directory G:\EFI\CLOVER\drivers\BIOS

The old HP server is set up with boot order USB device as first device.
When booting it tells to me, "Attempting to boot from USB"
pauses about 1/2 a second, says very briefly "6" and then the screen dies and the computer resets.

If I leave the USB in there it will repeat this cycle indefinitely.

What version of Clover would I use with an old XW8600?
In "boot disk utility" it only lists version 5140 through to 5149.

**Comment by Morbius01** [comment, marked as answer]
If I am correct, HP XW8600 was released in  ~2012-2013.

I use Clover on an even older motherboard (2009).
See my [post](https://github.com/CloverHackyColor/CloverBootloader/discussions/491).

I had difficulties to make my system work with recent versions of Clover.
For this reason, I use Clover v.4961 which perfectly works for me.

Hope the above was useful to you.

**Reply by KevinMoorman** [reply]
Nope the xw8600 dates back to 2007 (at least going from date codes on some of the BIOS systems that activate on bootup).
[Clover v.4961](https://web.archive.org/web/20180426065126/http://cvad-mac.narod.ru/index/bootdiskutility_exe/0-5) had the same crash/restart behavior as v.5149, but v.4861 was able to boot.
However, neither installation of Windows 10 was able to boot up (they came up as "Legacy HDD1" and "Legacy HDD2" -- huh?)
I wish there was some form of guide for configuring Clover but I can't seem to find any advice anywhere!

**Reply by Morbius01** [reply]
I am surprised that Clover v.4961 did not work for you as it did work for all other users who tried on my advice to use it. Have you used the Clover & BDU files which I have now posted [here](https://github.com/CloverHackyColor/CloverBootloader/discussions/491) ?

Well, if v.4861 works better for you, then you may like to try to use this version.

Usually, _‘Legacy HDDx’_ entries on the Clover Menu refer to previous installations of your OS. If you do not have anything else, it suggests that you have not installed Windows through Clover in the proper way .

I agree that there is a lack of serious and accurate guidelines on how to set up Clover. This is why, after having finally succeeded in setting up Clover and in installing various OSs, I am here (and on other forums too) trying to help others.

At this point, my suggestion for you on how to proceed would be :
1. Completely re-format your NVMe SSD.
2. Create a fresh Clover USB key with BDU, using either v.4961 or v.4861. Make sure that you copy the **EFI/CLOVER/drivers-Off/drivers64UEFI/NvmExpressDxe-64.efi** file to the **EFI/CLOVER/drivers64** and to the **EFI/CLOVER/drivers64UEFI** folders. The important point is that, with this key, you should be able to boot Clover and to access the Clover Menu. 
3. Once you have booted Clover, insert your Windows installation USB drive (which you should have created with the **Rufus Image Writer** utility from your Windows Installation .iso image). A new entry, libeled as something like _‘Boot UEFI Optical from …’_ or _'Boot Microsoft EFI cdboot from ...'_ (or similar) should appear. Select this entry, and see if it starts the Windows installation process. Through the installation process of Windows, Windows will reboot 2-3 times. At each reboot, you will need to boot Clover first, then select the same **new** entry from the Clover Menu which should be libeled as something like _'Boot Microsoft EFI Boot from EFI'_ or similar.

**Reply by KevinMoorman** [reply]
Ok! I booted into Clover v.4861 and then plugged in my second USB key outfitted with Win10 installer, and sure enough two new entries appeared: "Boot Win10 CD EFI" and "Boot Win10 EFI." The CD boot only crashed, and the Win10 boot displayed a "☺" character with blinking cursor and system hang (perplexing).
Anyway I was doing this over at my dad's house (it's his HP xw8600 server) but then he bought a second-hand Acer Aspire ATC-280 EB11 dated from 2014 and equipped with AMD A10-7800 "APU." He got that for $17 without HDD so we put in the M.2 NVME PCI-e card and the Clover v.4861 USB bootloader, worked like a charm.
Incidentally: we did not need to access the Win10 installer media from Clover on the Acer because the Win10 installer loads its own NVME driver and was able to install Windows without requiring BIOS or UEFI support. We only need Clover for booting into the Windows after it had been installed onto the NVME drive.
So thinking about that, I might try loading Windows on the HP xw8600 again but without trying to invoke the Win10 installer from the Clover bootloader.

**Reply by Morbius01** [reply]
If selecting the new entries which appeared after you inserted the Windows Installer drive makes the system crash (it happened to me too !), _try the following_.

After you insert the Installer drive, ignore the new entries in the Clover menu. Instead, go to **Exit** (the last selection on the Clover menu screen) **➔ Boot Maintenance Manager ➔ Boot from File ➔ Windows 10 USB Drive ➔ < efi > ➔ < boot > ➔ bootx64.efi**. Select it. _Does it start the installation process ?_

(Note that, since you use v.4861 (instead of the v.4961 I use), the names of the different sub-menus mentioned above might be slightly different.)
